packages/remark-iframes/src/index.js

```diff
@@ -1,60 +1,6 @@
 const {format, parse, URLSearchParams} = require('url')
-const http = require('http')
-const https = require('https')
-
-const makeHttpRequest = async url =>
```

Comment:
All of this was way too complicated. All we need here is to fetch a remote json object